xbmc/interfaces/python/PythonInvoker.cpp

```diff
@@ -36,6 +36,17 @@
 #include "platform/posix/XTimeUtils.h"
 #endif
 
+// This is a workaround to compile Kodi against python 3.8
+//! @todo implement a compliant way to get access to the chain of thread states
+#if PY_VERSION_HEX >= 0x03080000
+# define Py_BUILD_CORE
+# undef HAVE_STD_ATOMIC
+/* for access to the fields of PyInterpreterState */
+#  include "internal/pycore_pystate.h"
```

Comment:
include style is wrong, this in not a Kodi header

Reply:
thanx for the hint, updated.